elementary/cli/cli.py

```diff
@@ -27,6 +27,7 @@ def get_log_path(ctx):
         target_path = ctx_args[ctx_args.index(target_path_flag) + 1]
     finally:
         os.makedirs(os.path.abspath(target_path), exist_ok=True)
+        os.environ["DBT_LOG_PATH"] = os.path.abspath(target_path)
```

Comment:
the target_path isn't available from anywhere in the code since it only exist when we have the context and some functions initialize before this happens (for example I can't place it in the init of the dbtRunner because it's being imported before, and placing it inside the functions themselves doesn't feel fit) 
But you're right it doesn't really belong in this function 
I'm think of placing it in the config, but I need to make sure the config will be initialized before the dbt is running anywhere 👍